config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users do
+    resources :posts, only: %i[index show]
+  end
+  resources :users, only: %i[index show]
```

Comment:
- [ ] Let us avoid repetition here. The second routes for users can be used instead of the first one. We also want to avoid exposing unnecessary routes. Kindly refactor this instead.

  ```
  resources :users, only: %i[index show] do
    resources :posts, only: %i[index show]
  end